alphas/alpha_utils.py

```diff
@@ -684,6 +684,28 @@ def minus(data_a: pd.DataFrame, data_b: pd.DataFrame) -> pd.DataFrame:
         raise Exception("The summands are not the same datatypes and cannot be subtracted")
 
 
+def mult(
+    data_a: pd.DataFrame,
+    data_b: pd.DataFrame,
```

Comment:
could you remove the comma at the end so that the fxn definition is only on one line.

Reply:
The autocheckers are correcting me to make it two lines, for both cases

Reply:
nevermind i got it to work